### PR TITLE
Named entity recognition

### PR DESCRIPTION
**Pull request by ngarneau**
Hey guys, on the same idea as the POS tagger, here is the NER. As @etrain said, it would be better if the Keystone API wouldn't expose Epic's interfaces so I'll work on this as a next step.

Let me know if you see anything here, I built it pretty much the same way I did with the POS Tagger.

Thanks,
Nicolas

### CODE REVIEW

File: src/main/scala/nodes/nlp/NER.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+  * Transformer that uses Epic's named entity recognition on a sequence of words.
+  * It is recommended to use a Tokenizer before applying the NER,
+  * assuming we get a list of strings as input.
+  *
+  * Here's an example:
+  * {{{
+  *   val model = epic.models.NerSelector.loadNer("en").get
+  *   val NERmodel = NERModel(model).apply(data)
```
Comment (etrain) [inline src/main/scala/nodes/nlp/NER.scala]: this docstring looks broken?

File: src/main/scala/nodes/nlp/NER.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  * }}}
+  *
+  * @param model The NER model loaded from the Epic library
+  */
+case class NER(@transient model: SemiCRF[Any, String])
```
Comment (etrain) [inline src/main/scala/nodes/nlp/NER.scala]: Is `Any` appropriate here? 
why not `case class NER[T](@transient model: SemiCRF[T, String]) extends Transformer[Array[String], Segmentation[T, String]]`?

**Comment by etrain** [comment]
A couple of minor comments - in general we're trying to preserve type-safety at the level of pipeline nodes, and I think a generic will solve that problem here (though it may not - the epic type system isn't as crazy as breeze's but it's not the simplest one either)

**Comment by ngarneau** [comment]
I definitely agree with your comments.

**Comment by etrain** [comment]
LGTM. Thanks!

**Pull request merged**